### PR TITLE
print targets one per line

### PR DESCRIPTION
**Pull request by Habbie**

### CODE REVIEW

**Comment by Habbie** [comment]
Because at least for pdns.git, all items on one line is becoming unreadable.

**Comment by omoerbeek** [comment]
Sound like a job for `fold -s` to me...

**Comment by Habbie** [comment]
> Sounds like a job for `fold -s` to me...

I don't understand, what do you mean?

**Comment by omoerbeek** [comment]
> > Sounds like a job for `fold -s` to me...
> 
> I don't understand, what do you mean?

```
targets=$(ls $BUILDER_SUPPORT_ROOT/dockerfiles/Dockerfile.target.* | sed 's/.*Dockerfile.target.//' | tr '\n' ' ' | fold -s)
```
is nicer imo, as it does not use many lines but still avoids long lines.

**Comment by Habbie** [comment]
```
amazon-2 centos-7 centos-7-amd64 centos-8 centos-8-amd64 centos-8-arm64 
debian-buster debian-buster-amd64 debian-buster-arm64 docs raspbian-buster 
sdist ubuntu-bionic ubuntu-bionic-amd64 ubuntu-bionic-arm64 ubuntu-focal 
ubuntu-focal-amd64 ubuntu-focal-arm64
```

I find this very hard to read compared to:
```
  amazon-2
  centos-7
  centos-7-amd64
  centos-8
  centos-8-amd64
  centos-8-arm64
  debian-buster
  debian-buster-amd64
  debian-buster-arm64
  docs
  raspbian-buster
  sdist
  ubuntu-bionic
  ubuntu-bionic-amd64
  ubuntu-bionic-arm64
  ubuntu-focal
  ubuntu-focal-amd64
  ubuntu-focal-arm64
```

**Comment by omoerbeek** [comment]
ok, i have no strong feelings here...

File: build.sh [hunk 1]
```diff
@@ -110,6 +109,8 @@ usage() {
     echo "  -k URL          - Use URL as the cache for kaniko layers."
     echo
     echo "Targets:  $targets"
```
Comment (wojas) [inline build.sh:111]:
```suggestion
    echo "Targets:"
```

**Pull request merged**